src/VecSim/spaces/IP/IP_SVE_FP32.h

```diff
@@ -62,7 +62,7 @@ float FP32_InnerProductSIMD_SVE(const void *pVect1v, const void *pVect2v, size_t
     sum0 = svadd_f32_z(svptrue_b32(), sum0, sum1);
     sum2 = svadd_f32_z(svptrue_b32(), sum2, sum3);
```

Comment:
Here and in other files as well
```suggestion
    sum0 = svadd_f32_x(svptrue_b32(), sum0, sum1);
    sum2 = svadd_f32_x(svptrue_b32(), sum2, sum3);
```